recipes/logr/v0.7/conanfile.py

```diff
@@ -0,0 +1,136 @@
+    )
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ngrodzitski/logr"
+    topics = ("logger", "development", "util", "utils")
+    settings = "os", "arch", "compiler", "build_type"
+    package_type = "header-library"
+    options = {
+        "with_spdlog_backend": [True, False],
+        "with_glog_backend": [True, False],
+        "with_log4cplus_backend": [True, False],
+        "with_boostlog_backend": [True, False],
+    }
+    default_options = {
+        "with_spdlog_backend": True,
+        "with_glog_backend": False,
+        "with_log4cplus_backend": False,
+        "with_boostlog_backend": False,
+    }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("fmt/10.2.1")
+
+        if self.options.with_spdlog_backend:
+            self.requires("spdlog/1.12.0")
+
+        if self.options.with_glog_backend:
+            self.requires("glog/0.6.0")
+
+        if self.options.with_log4cplus_backend:
+            self.requires("log4cplus/2.1.0")
+
+        if self.options.with_boostlog_backend:
+            self.requires("boost/1.83.0")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        minimal_cpp_standard = "17"
+        if self.settings.get_safe("compiler.cppstd"):
+            check_min_cppstd(self, minimal_cpp_standard)
+        minimal_version = {
+            "gcc": "10",
+            "clang": "11",
+            "apple-clang": "12",
+        }
+        check_min_vs(self, 192)
+        if not is_msvc(self):
+            minimum_version = minimal_version.get(str(self.settings.compiler), False)
+            if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.ref} requires minimum {self.settings.compiler}-{minimum_version}."
+                )
+
+    def build(self):
+        pass
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+                destination=self.source_folder, strip_root=True)
+
+    def package(self):
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "*.*pp", src=os.path.join(self.source_folder, "logr", "include"), dst=os.path.join(self.package_folder, "include"))
+
+        if not self.options.with_spdlog_backend:
+            rm(self, f"include/logr/spdlog_backend.hpp", self.package_folder)
+
+        if not self.options.with_glog_backend:
+            rm(self, f"include/logr/glog_backend.hpp", self.package_folder)
+
+        if not self.options.with_log4cplus_backend:
+            rm(self, f"include/logr/log4cplus_backend.hpp", self.package_folder)
+
+        if not self.options.with_boostlog_backend:
+            rm(self, f"include/logr/boostlog_backend.hpp", self.package_folder)
```

Comment:
```suggestion
        include_folder = os.path.join(self.package_folder, "include", "logr")
        if not self.options.with_spdlog_backend:
            rm(self, "spdlog_backend.hpp", include_folder)

        if not self.options.with_glog_backend:
            rm(self, "glog_backend.hpp", include_folder)

        if not self.options.with_log4cplus_backend:
            rm(self, "log4cplus_backend.hpp", include_folder)

        if not self.options.with_boostlog_backend:
            rm(self, "boostlog_backend.hpp", include_folder)
```

f-string is not required here.